sdk/python/tests/compiler/testdata/loop_static_with_parallelism.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  annotations:
+    pipelines.kubeflow.org/pipeline_spec: '{"inputs": [{"default": "10", "name": "my_pipe_param",
+      "optional": true}], "name": "para-loop-pipeline"}'
+    sidecar.istio.io/inject: 'false'
+    tekton.dev/artifact_bucket: mlpipeline
+    tekton.dev/artifact_endpoint: minio-service.kubeflow:9000
+    tekton.dev/artifact_endpoint_scheme: http://
+    tekton.dev/artifact_items: '{"para-loop-inner-op1": [], "para-loop-inner-op2":
+      [], "para-loop-out-op": []}'
+    tekton.dev/input_artifacts: '{}'
+    tekton.dev/output_artifacts: '{}'
+  name: para-loop-pipeline
+spec:
+  params:
+  - name: my_pipe_param
+    value: '10'
+  pipelineSpec:
+    params:
+    - default: '10'
+      name: my_pipe_param
+    tasks:
+    - name: para-loop-out-op
+      params:
+      - name: my_pipe_param
+        value: $(params.my_pipe_param)
+      taskSpec:
+        metadata:
+          annotations:
+            tekton.dev/template: ''
+          labels:
+            pipelines.kubeflow.org/cache_enabled: 'true'
+            pipelines.kubeflow.org/generation: ''
+            pipelines.kubeflow.org/pipelinename: ''
+        params:
+        - name: my_pipe_param
+        steps:
+        - args:
+          - echo $(inputs.params.my_pipe_param)
+          command:
+          - sh
+          - -c
+          image: library/bash:4.4.23
+          name: main
+      timeout: 0s
+    - name: para-loop-pipeline-for-loop-2
+      parallelism: 5
```

Comment:
I don't think the `parallelism` is valid over here. Although the Tekton controller will ignore it for now, it might cause problems in the future if Tekton check this field.

Reply:
OK, I will remove this from the pipelinerun yaml.